.gitmodules

```diff
@@ -70,3 +70,6 @@
 [submodule "auxil/zeek-af_packet-plugin"]
 	path = auxil/zeek-af_packet-plugin
 	url = https://github.com/zeek/zeek-af_packet-plugin.git
+[submodule "auxil/libunistd"]
+	path = auxil/libunistd
+	url = https://github.com/zeek/libunistd
```

Comment:
The license for libunistd needs to be added to COPYING.3rdparty.

Reply:
done